testing/e2e/infra/provision.go

```diff
@@ -44,7 +44,7 @@ func (i *infra) Provision(ctx context.Context, tenantId, subscriptionId, applica
 	}
 
 	var err error
-	ret.ResourceGroup, err = clients.NewResourceGroup(ctx, subscriptionId, i.ResourceGroup, i.Location, clients.DeleteAfterOpt(6*time.Hour))
+	ret.ResourceGroup, err = clients.NewResourceGroup(ctx, subscriptionId, i.ResourceGroup, i.Location, clients.DeleteAfterOpt(9*time.Hour))
```

Comment:
nit can we bump to 24 or 18? I enjoy queueing builds at EOD and seeing what they look like the next morning

Reply:
sure. will bump after cleanup pr merged